### PR TITLE
Update monero-core_de.ts

### PR DESCRIPTION
**Pull request by hundehausen**
I added some missing translation. I think it is a good idea if someone would read my corrections and my added translation one more time, so we can be sure everything is right.

### CODE REVIEW

**Comment by jonathancross** [comment]
Hi @hundehausen, can you please take a look at #734?
Would be great to get that merged, then add any required fixes to that.
(we have a bit of a traffic jam with translations currently)

File: translations/monero-core_de.ts [hunk 1]
```diff
@@ -993,7 +993,7 @@
     <message>
         <location filename="../pages/Settings.qml" line="258"/>
         <source>Username</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Benutzername</translation>
```
Comment (jonathancross) [inline translations/monero-core_de.ts]: Note: we should be removing `type="unfinished"` when a translation is finished.

**Comment by jonathancross** [comment]
Hi @hundehausen, now that #734 has been merged, much of this Pull Request and comments will be obsolete.
I'd suggest closing this PR and making any required changes next week or so after we have a fresh, updated set of strings to translate (see #736).

**Comment by jonathancross** [comment]
@hundehausen or @fluffypony - Please close this.
It is now far out of date and will get worse with #783.

**Comment by fluffypony** [comment]
Closed by request

**Pull request closed without merging**